File: scholar-web/src/main/webapp/app/model/student/lookup/LookupSearch.js
Ext.define('scholar.model.student.lookup.LookupSearch', {
	extend: 'Ext.data.Model',
	idProperty:'admissionId',
	fields : [ {
		name : 'admissionId',
		type : 'int'
	},{
		name : 'admissionNumber',
		type : 'string'
	}, {
		name : 'studentName',
		type : 'string'
	}, {
		name : 'standard',
		type : 'string'
	}, {
		name : 'address',
		type : 'string'
	}, {
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ]	
});

File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/LookupSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class LookupSearch extends AbstractEntity {
	int admissionId;
	String admissionNumber;
	String studentName;
	String standard;
	String address;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}
